book/quick_tour.md

```diff
@@ -1,53 +1,227 @@
+# => ╰───┴─────────────────┴──────┴───────────┴───────────────╯
+```
+
+## More Than Just Directories
+
+Of course, this isn't limited to the `ls` command. Nushell follows the Unix philosophy where each command does one thing well and you can typically expect the output of one command to become the input of another. This allows us to mix-and-match commands in many different combinations.
+
+Let's look at a different command:
+
+```nu:no-line-numbers
+ps
+# => ╭───┬──────┬──────┬───────────────┬──────────┬──────┬───────────┬─────────╮
+# => │ # │ pid  │ ppid │     name      │  status  │ cpu  │    mem    │ virtual │
+# => ├───┼──────┼──────┼───────────────┼──────────┼──────┼───────────┼─────────┤
+# => │ 0 │    1 │    0 │ init(void)    │ Sleeping │ 0.00 │   1.2 MiB │ 2.2 MiB │
+# => │ 1 │    8 │    1 │ init          │ Sleeping │ 0.00 │ 124.0 KiB │ 2.3 MiB │
+# => │ 2 │ 6565 │    1 │ SessionLeader │ Sleeping │ 0.00 │ 108.0 KiB │ 2.2 MiB │
+# => │ 3 │ 6566 │ 6565 │ Relay(6567)   │ Sleeping │ 0.00 │ 116.0 KiB │ 2.2 MiB │
+# => │ 4 │ 6567 │ 6566 │ nu            │ Running  │ 0.00 │  28.4 MiB │ 1.1 GiB │
+# => ╰───┴──────┴──────┴───────────────┴──────────┴──────┴───────────┴─────────╯
+```
+
+You may be familiar with the Linux/Unix `ps` command. It provides a list of all of the current processes running in the system along with their current status. As with `ls`, Nushell provides a cross-platform, built-in [`ps` command](/commands/docs/ps.md) that returns its results as structured data.
+
+::: note
+The traditional Unix `ps` only shows the current process and its parents by default. Nushell's implementation shows all of the processes on the system by default.
+:::
+
+What if we wanted to just show the processes that are actively running? As with `ls` above, we can also work with the table that `ps` _outputs_:
+
+```nu
+ps | where status == Running
+# => ╭───┬──────┬──────┬──────┬─────────┬──────┬──────────┬─────────╮
+# => │ # │ pid  │ ppid │ name │ status  │ cpu  │   mem    │ virtual │
+# => ├───┼──────┼──────┼──────┼─────────┼──────┼──────────┼─────────┤
+# => │ 0 │ 6585 │ 6584 │ nu   │ Running │ 0.00 │ 31.9 MiB │ 1.2 GiB │
+# => ╰───┴──────┴──────┴──────┴─────────┴──────┴──────────┴─────────╯
+```
+
+::: tip
+Remember above, where the `size` column from the `ls` command was a `filesize`? In this case, `status` may look like an "enum" type from some programming languages,
+but here it's really a normal `string`.
+
+You can examine the types for the table's columns using:
+
+```nu
+ps | describe
+# => table<pid: int, ppid: int, name: string, status: string, cpu: float, mem: filesize, virtual: filesize> (stream)
+```
+
+:::
+
+## Command Arguments in a Pipeline
+
+Sometimes, a command takes an _argument_ instead of pipeline _input_. For this scenario, Nushell provides the [`$in` variable](./pipelines.md#pipeline-input-and-the-special-in-variable) that let's you use the previous command's output in variable-form. For example:
```

Comment:
I was wondering if it makes sense to give a heuristic for when we don't use input here like:
```suggestion
Sometimes, a command takes an _argument_ instead of pipeline _input_. This is for example the case for commands where two arguments have different meanings, like the source and destination path in `cp`. For this scenario, Nushell provides the [`$in` variable](./pipelines.md#pipeline-input-and-the-special-in-variable) that let's you use the previous command's output in variable-form. For example:
```

Reply:
Agreed - I considered this.  Let me see how it looks as a callout.  Will update sometime today, hopefully. Thanks!